sdk/cognitivelanguage/azure-ai-language-conversations/README.md

```diff
@@ -67,6 +67,36 @@ credential = AzureKeyCredential("<api-key>")
 client = ConversationAuthoringClient(endpoint, credential)
 ```
 
+#### Create a client with an Azure Active Directory Credential
+
+To use an [Azure Active Directory (AAD) token credential][cognitive_authentication_aad],
+provide an instance of the desired credential type obtained from the
+[azure-identity][azure_identity_credentials] library.
+Note that regional endpoints do not support AAD authentication. Create a [custom subdomain][custom_subdomain]
+name for your resource in order to use this type of authentication.
+
+Authentication with AAD requires some initial setup:
+
+- [Install azure-identity][install_azure_identity]
+- [Register a new AAD application][register_aad_app]
+- [Grant access][grant_role_access] to the Language service by assigning the "Cognitive Services User" role to your service principal.
```

Comment:
Actually, is it "Cognitive Services User" or "Cognitive Services Language Reader"? @ChongTang?

Reply:
Not sure, but want to merge this PR to get the resulting APIView approved. Will open a separate PR to fix this once I find out.